### PR TITLE
kola/test/metadata: add coreos-metadata

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: kola/tests/metadata/contents.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		                   ]
+		               }
+		           }`,
+	})
+
+	register.Register(&register.Test{
+		Name:        "coreos.metadata.azure",
+		Run:         verifyAzure,
+		ClusterSize: 1,
+		Platforms:   []string{"azure"},
```
Comment (mischief) [inline kola/tests/metadata/contents.go]: we don't even support azure :C

File: kola/tests/metadata/contents.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"github.com/coreos/mantle/platform"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "coreos.metadata.aws",
+		Run:         verifyAWS,
+		ClusterSize: 1,
+		Platforms:   []string{"aws"},
+		UserData: `{
```
Comment (mischief) [inline kola/tests/metadata/contents.go]: looks like you could dedup these two userdatas?

**Comment by pbx0** [comment]
lgtm

File: kola/tests/metadata/contents.go (hunk 1 above)
Comment (pbx0) [inline kola/tests/metadata/contents.go]: This will essentially just disable the test for now.

**Pull request merged**